content/en/docs/what-is-opentelemetry.md

```diff
@@ -1,52 +1,61 @@
 ---
 title: What is OpenTelemetry?
-description: A short explanation of what OpenTelemetry is, and is not.
+description: A short explanation of what OpenTelemetry is and isn't.
 aliases: [/about, /docs/concepts/what-is-opentelemetry, /otel]
 weight: -1
 ---
 
 OpenTelemetry is an
 [Observability](/docs/concepts/observability-primer/#what-is-observability)
-framework and toolkit designed to create and manage _telemetry data_ such as
+framework and toolkit designed to create and manage telemetry data such as
 [traces](/docs/concepts/signals/traces/),
 [metrics](/docs/concepts/signals/metrics/), and
 [logs](/docs/concepts/signals/logs/). Crucially, OpenTelemetry is vendor- and
 tool-agnostic, meaning that it can be used with a broad variety of Observability
-backends, including open source tools like
+back ends, including open source tools like
 [Jaeger](https://www.jaegertracing.io/) and
 [Prometheus](https://prometheus.io/), as well as commercial offerings.
-OpenTelemetry is a
-[Cloud Native Computing Foundation (CNCF)](https://www.cncf.io) project.
 
-## Why OpenTelemetry?
+OpenTelemetry is not an observability back end like Jaeger, Prometheus, or other
+commercial vendors. OpenTelemetry is focused on the generation, collection,
+management, and export of telemetry data. The storage and visualization of that
+data is intentionally left to other tools.
```

Comment:
```suggestion
OpenTelemetry is not an observability back end like Jaeger, Prometheus, or other
commercial vendors. OpenTelemetry is focused on the generation, collection,
management, and export of telemetry. A major goal of OpenTelemetry is that you
can easily instrument your applications or systems, no matter their language, infrastructure,
or runtime environment. Crucially, the storage and visualization of telemetry is intentionally
left to other tools.
```

Reply:
Good point, I always meant to ask about "telemetry data" - we use this expression elsewhere and I feel it's redundant.